README.md

```diff
@@ -17,3 +17,26 @@ tool that bpftrace maintainers will help support over long periods of time.
 
 
 [0]: https://github.com/bpftrace/bpftrace/blob/master/CONTRIBUTING-TOOLS.md
+
+## Licensing
+
+Depending on which [BPF helpers](kernel-versions.md#helpers) are used by 
+bpftrace when compiling your program, a GPL-compatible license is required for 
+bpftrace programs/tools.
+
+You can set the license as a comment in your source code (at the top of the file). Note: it supports multiple words and quotes are not necessary. **If the license is not specified, bpftrace will automatically define the license of the program as GPL v2.**
+
+Examples:
+
+- `// SPDX-License-Identifier: GPL-2.0+`
```

Comment:
"GPL-2.0+" is a deprecated license ID: https://spdx.org/licenses/
Better to reference the replacement, "GPL-2.0-or-later", instead